specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.6-preview.1/backuprestore.json

```diff
@@ -363,6 +489,27 @@
         "storageResourceUri"
       ]
     },
+    "PreBackupOperationParameters": {
+      "properties": {
+        "storageResourceUri": {
+          "type": "string",
+          "description": "Azure Blob storage container Uri",
+          "x-nullable": true
```

Comment:
Remove all x-nullable. It's not the opposite of required parameters/properties. It means the value can be an explicit `null` in JSON, which really only has meaning for JSON merge-patch (PATCH operations).